internal/tools/pipeline/providers/pipeline/create_v2.go

```diff
@@ -766,6 +771,21 @@ func (s *pipelineService) GetYmlActionTasks(pipelineYml *pipelineyml.PipelineYml
 	return actionTasks
 }
 
+func (s *pipelineService) tryGetOrgName(p *spec.Pipeline) string {
+	orgIDStr := p.MergeLabels()[apistructs.LabelOrgID]
+	orgID, err := strconv.ParseUint(orgIDStr, 10, 64)
+	if err != nil {
+		s.p.Log.Warnf("failed to parse orgID: %s, err: %v", orgIDStr, err)
```

Comment:
adjust log level, it will cause too much useless logs

Reply:
adjusted to `debug` level